keps/sig-storage/1698-generic-ephemeral-volumes/kep.yaml

```diff
@@ -5,7 +5,7 @@ authors:
 owning-sig: sig-storage
 participating-sigs:
   - sig-storage
-status: implementable
+status: implemented
```

Comment:
This is assuming that the actual API change for GA gets in. Do we keep this PR open until that is done or shall I revert to "implementable" and later change to "implemented"?

Reply:
Revert to implementable.

We mark as implemented after the release with all the changes are actually released (so once 1.23 is out).

Reply:
Done.